### PR TITLE
fix: add skip domain injection description

### PR DESCRIPTION
**Pull request by AtofStryker**
- Closes N/A

### User facing changelog

Adds locales for `experimentalSkipDomainInjection`
### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/graphql/schemas/cloud.graphql [hunk 1]
```diff
@@ -1,6 +1,72 @@
 ### This file was generated by Nexus Schema
 ### Do not make changes to this file directly
 
+"""
```
Comment (mjhenkes) [inline packages/graphql/schemas/cloud.graphql]: Whats with the graph ql schema changes?

File: packages/frontend-shared/src/locales/en-US.json [hunk 2]
```diff
@@ -522,6 +522,10 @@
       "experimentalOriginDependencies": {
         "name": "Origin Dependencies",
         "description": "Enables support for `require`/`import` within `cy.origin`."
+      },
+      "experimentalSkipDomainInjection": {
+        "name": "Skip Domain Injection",
+        "description": "Disables setting `document.domain` to the applications super domain on injection"
```
Comment (mschile) [inline packages/frontend-shared/src/locales/en-US.json]:
```suggestion
        "description": "Disables setting `document.domain` to the application's super domain on injection"
```

File: packages/graphql/schemas/cloud.graphql (hunk 1 above)
Comment (AtofStryker) [reply packages/graphql/schemas/cloud.graphql]: someone made schema changes and didn't run `yarn`

File: packages/frontend-shared/src/locales/en-US.json (hunk 2 above)
Comment (AtofStryker) [reply packages/frontend-shared/src/locales/en-US.json]: fixed in b80086a7ba

**Review by mjhenkes — APPROVED** [review]

File: packages/graphql/schemas/cloud.graphql (hunk 1 above)
Comment (AtofStryker) [reply packages/graphql/schemas/cloud.graphql]: going to remove these as checking them in causes typing issues

Comment (AtofStryker) [reply packages/graphql/schemas/cloud.graphql]: fixed in a8808634a3

**Review by mschile — APPROVED** [review]

**Review by emilyrohrbough — APPROVED** [review]

**Pull request merged**